docs/components/movement-sensor/gps/set-up-base-station.md

```diff
@@ -0,0 +1,239 @@
+
+- Default: **Portable**
+- Keep the default selection of **Portable** here.
+
+**Min SV Elevation:** Minimum elevation in degrees for a GNSS satellite to be used in a fix.
+
+- Select **10**.
+
+**Min C/N0**: Minimum satellite signal level to use in a fix.
+
+- Default: **6dBHz** for ZED-F9P and **20dBHz** for ZED-F9R.
+- Keep the default value here.
+
+**Constellations:** The receiver is capable of concurrently receiving signals from multiple satellites across multiple constellations.
+Some applications dictate the need to turn off certain constellation reception.
+Deselecting a given constellation will cause the receiver to ignore those signals preventing them from being used during position calculations.
+
+- Default: **GPS**, **SBAS**, **GLONASS**, **BeiDou**, and **Galileo**.
+- Keep the default selections here.
+
+With these recommended values selected, your GNSS Configuration should look like this:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-GNSS-config.png" alt="Recommended GNSS Configuration." resize="900x" style="width:450px" >}}
+
+Now click on the **Message Rates** box.
+In the Message Menu make sure to enable the following messages, because VIAM’s RTK driver uses these messages:
+
+- GGA
+- GLL
+- GNS
+- GSV
+- RMC
+- VTG
+
+Make your **Message Rates** option look like this:
+
+{{< imgproc src="/components/movement-sensor/sparkfun-GNSS-message-rates.png" alt="Recommended GNSS Message Rates Configuration." resize="900x" style="width:450px" >}}
+
+### Update Base configuration
+
+Select the **Base Configuration** dropdown on the RTK setup configure page.
+
+**Survey-In:** If the precise location of a base station is not known it may be obtained by ‘surveying’ the location.
+The base is fixed in one place and takes approximately 60 seconds worth of readings to obtain a best fit location based on the measurements. This method achieves ~30cm accurate position but can vary.
+Increasing the minimum observation time or required mean deviation increases accuracy but only to a point.
+Better accuracy is achieved with long-term logging and post processing.
+
+- Default: 60s and 5.0m.
+- Limits: 60 to 900s, 1.0 to 5.0m.
+- Enable the **Survey-In** mode, and configure it as follows:
+  - **Minimum observation time**: 60 (default)
+  - **Required Mean 3D standard deviation (m)**: 5.00 (default)
+
+**Fixed**: You only want the **Fixed** mode if your reference station will be stationary at a permanent location.
+
+- Keep this deselected.
+
+**Enable NTRIP Server:** You use this NTRIP server to upload the base's correction data to the casting service.
+Other devices can then access the correction data using an NTRIP client.
+
+- Default: Disabled.
+- Enable this option. Enter the following credentials:
+  - **Caster Host:** rtk2go.com
+  - **Caster Port:** 2101
+  - **Caster User:** <test@test.org>
```

Comment:
@susmitaSanyal Should I instruct them to put in their own emails here? 

MD linter won't let me write test@test.org without the carrots due to empty url rule

Reply:
yes, users can use their own email. I didn't know what to put so I used test lol

Reply:
Ok sweet thanks!